exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -176,7 +195,7 @@ def _translate_to_zipkin(self, spans: Sequence[Span]):
                 "timestamp": start_timestamp_mus,
                 "duration": duration_mus,
                 "localEndpoint": local_endpoint,
-                "kind": SPAN_KIND_MAP[span.kind],
+                "kind": SPAN_KIND_MAP[TRANSPORT_FORMAT_JSON][span.kind],
```

Comment:
Looks like a pointless lookup. Could use a root level `SPAN_KIND_MAP_JSON` dictionary.

Reply:
To me this kind of usage implies that the key is dynamic and may change between executions which clearly is not the case and can be misleading when reading the code. IMO using a root level map where the function will always use the exact same map reads a lot better.